docs/guides/commands.md

```diff
@@ -127,40 +127,43 @@ When run as a `kn` plugin.
 kn func delete <name> [-n namespace, -p path]
 ```
 
-## `emit`
-
-Emits a CloudEvent, sending it to the deployed function. The user may specify the event type, source and ID,
-and may provide event data on the command line or in a file on disk. By default, `event` works on the local
-directory, assuming that it is a function project. Alternatively the user may provide a path to a project
-directory using the `--path` flag, or send an event to an arbitrary endpoint using the `--sink` flag. The
-`--sink` flag also accepts the special value `local` to send an event to the function running locally, for
-example, when run via `func run`.
+## `invoke`
+
+Invokes a function. By default, a locally running instance will be preferred
```

Comment:
Maybe "Invokes a __running__ function."?

Reply:
Yes, that is better.  Modification made.

Tangential:  I think this entire document could perhaps be auto-generated by the --help text from the commands themselves.  After all, the two locations seem to be serving both the same purpose and same audience.

That would of course require some hackery, but it might be worth the effort.